modules/tinymce/CHANGELOG.md

```diff
@@ -8,113 +8,134 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The object returned from the `editor.fire()` API was incorrect if the editor had been removed #TINY-8018
 - The `editor.selection.getContent()` API did not respect the `no_events` argument #TINY-8018
+- The `editor.annotator.remove` API did not keep selection when removing the annotation #TINY-8195
 - The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API #TINY-8018
+- The `beforeinput` and `input` events would sometimes not fire as expected when deleting content #TINY-8168 #TINY-8329
 - The `table` plugin would sometimes not correctly handle headers in the `tfoot` section #TINY-8104
-- The aria labels for the color picker dialog were not translated #TINY-8381
-- The `editor.annotator.remove` did not keep selection when removing the annotation #TINY-8195
 - The `silver` theme UI was incorrectly rendered before plugins had initialized #TINY-8288
+- The aria labels for the color picker dialog were not translated #TINY-8381
+- Fixed sub-menu items not read by screen readers. Patch contributed by westonkd #TINY-8417
 - Dialog labels and other text-based UI properties did not escape HTML markup #TINY-7524
-- Deleting content would sometimes not fire `beforeinput` and `input` events as expected #TINY-8168 #TINY-8329
 - Anchor elements would render incorrectly when using the `allow_html_in_named_anchor` option #TINY-3799
-- Fixed sub-menu items not read by screen readers. Patch contributed by westonkd #TINY-8417
 - The Home or End keys would move out of a editable element contained within a noneditable element #TINY-8201
 
 ### Removed
 - Removed the jQuery integration #TINY-4518
+- Removed the `imagetools` plugin, which is now classified as a Premium plugin #TINY-8209
+- Removed the `imagetools` dialog component #TINY-8333
+- Removed the `toc` plugin, which is now classified as a Premium plugin #TINY-8250
+- Removed the `tabfocus` plugin #TINY-8315
+- Removed the `textpattern` plugin's API as part of moving it to core #TINY-8312
+- Removed the `table` plugin's API #TINY-8273
+- Removed the callback for the `EditorUpload` API #TINY-8325
+- Removed the `filterNode` method from the `DomParser` API #TINY-8249
+- Removed the `SaxParser` API #TINY-8218
+- Removed the `tinymce.utils.Promise` API #TINY-8241
+- Removed the `toHex` function for the `DOMUtils` and `Styles` APIs #TINY-8163
+- Removed the `execCommand` handler function from the plugin and theme interfaces #TINY-7829
+- Removed the `editor.settings` property as it has been replaced by the new Options API #TINY-8236
+- Removed the `shortEnded` and `fixed` properties on `tinymce.html.Node` class #TINY-8205
+- Removed the `mceInsertRawHTML` command #TINY-8214
+- Removed the style field from the `image` plugin dialog advanced tab #TINY-3422
+- Removed the legacy `mobile` theme #TINY-7832
+- Removed the legacy browser detection properties from `Env` #TINY-8162
+- Removed support for Microsoft Internet Explorer 11 #TINY-8194 #TINY-8241
```

Comment:
Should this be up the top since it's one of the key things we need to highlight?

Reply:
I'd be fine with that. Can do

Reply:
Done